webviz_subsurface/plugins/_parameter_analysis/views/selector_view.py

```diff
@@ -304,11 +305,9 @@ def filter_vector_selector(
     )
 
 
-def color_selector(parent, tab: str, px_colors: dict = None, height=None):
-    custom_colors = {
-        "theme": parent.theme.plotly_theme.get("layout", {}).get("colorway", {})[1:12]
-    }
-
+def color_selector(
+    parent, tab: str, colors: list = None, bargap: float = None, height: float = None
```

Comment:
Might have to
`from typing import Optional`

Mismatch between type hint and a default `None` value are recurring a few other places as well.
`some_variable: some_type = None`
are recurring quite a few places.
```suggestion
    parent, tab: str, colors: Optional[list] = None, bargap: Optional[float] = None, height: Optional[float] = None
```

Reply:
I wasn't aware of this Optional, thanks 👍